rs/protobuf/def/registry/subnet/v1/subnet.proto

```diff
@@ -317,8 +318,11 @@ message ChainKeyConfig {
   // Signature requests will timeout after the given number of nano seconds.
   optional uint64 signature_request_timeout_ns = 2;
   // Key rotation period of a single node in milliseconds.
-  // If none is specified key rotation is disabled.
+  // If none is specified, key rotation is disabled.
   optional uint64 idkg_key_rotation_period_ms = 3;
+  // Maximum number of pre-signature transcripts that can be worked on in
+  // parallel to fill the pre-signature stash.
+  optional uint32 max_pre_signature_transcripts_in_creation = 4;
```

Comment:
This name is confusing.  I had to read the description to understand how it was different than `pre_signatures_to_create_in_advance`

Could you call it max_parallel_pre_signature_generations or something like that?